public/app/features/explore/TypeaheadItem.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FunctionComponent, useContext } from 'react';
+// @ts-ignore
+import Highlighter from 'react-highlight-words';
+import { css, cx } from 'emotion';
+import { GrafanaTheme, ThemeContext, selectThemeVariant } from '@grafana/ui';
+
+import { CompletionItem } from 'app/types/explore';
+
+interface Props {
+  isSelected: boolean;
+  item: CompletionItem;
+  onClickItem: (suggestion: CompletionItem) => void;
+  prefix?: string;
+  style: any;
+}
+
+const getStyles = (theme: GrafanaTheme) => ({
+  typeaheadItem: css`
+    height: auto;
```

Comment:
FYI for easier debugging it's wise to add `label: component-name;` to css. Thanks to that generated class name will have `component-name` in it and it's easier to reason where the styles come from.

Reply:
Thanks, I'm still not a huge fan of the 
```typescript 
css`...`
```
syntax, it takes some getting used to.